jekyll-docs/photon.md

```diff
@@ -2,7 +2,7 @@
 title: Photon Platform Integration
 ---
 ## Photon Platform: Introduction
-Docker volumes on [Photon platform](https://vmware.github.io/photon-controller) are provisioned, managed entirely via the Photon platform [API](https://github.com/vmware/photon-controller/wiki/API). Docker volumes are placed on datastores as decided by the Photon platform. The Photon platform selects datastores from within those that have been explicitly configured by the user on hosts managed by the Photon Controller and include backends such as NFS, SAN, VSAN, vVol. Accessibility of volumes to VMs running on the hosts is therefore dependent on whether the datastores are shared among hosts configured to the Photon platform. Accessibility of docker volumes on Photon is governed by the tenancy model of Photon, where each volume is scoped to a project within a tenant. Volumes are scoped to a project and hence accessible to all VMs within the same project. In addition, the count of volumes and storage capacity allocated (in total) to docker volumes within a project are managed via resource limits defined by the resource ticket associated with the tenant that the project belongs to. The Photon platform admin must hence take care to configure sufficient storage capacity, taking into consideration the max. number of volumes and the typical sizes of those. Once a configuration has been created, the Photon platform seamlessly manages accounting the count and storage capacity consumed by volumes in a project.
+Docker volumes on [Photon platform](https://vmware.github.io/photon-controller) are provisioned, managed entirely via the Photon platform [API](https://github.com/vmware/photon-controller/wiki/API). Docker volumes are placed on datastores as decided by the Photon platform. The Photon platform selects datastores from within those that have been explicitly configured by the user on hosts managed by the Photon Controller and include backends such as NFS, SAN, vSAN, vVol. Accessibility of volumes to VMs running on the hosts is therefore dependent on whether the datastores are shared among hosts configured to the Photon platform. Accessibility of docker volumes on Photon is governed by the tenancy model of Photon, where each volume is scoped to a project within a tenant. Volumes are scoped to a project and hence accessible to all VMs within the same project. In addition, the count of volumes and storage capacity allocated (in total) to docker volumes within a project are managed via resource limits defined by the resource ticket associated with the tenant that the project belongs to. The Photon platform admin must hence take care to configure sufficient storage capacity, taking into consideration the max. number of volumes and the typical sizes of those. Once a configuration has been created, the Photon platform seamlessly manages accounting the count and storage capacity consumed by volumes in a project.
```

Comment:
I'd given some comments earlier on Photon and listing here again:

1. Please mention that VSAN is the only storage backend for the photon platform (not NFS/vmfs/vvol).
1. Photon doesn't support the user to configure datastores, instead photon manages allocations based on disk storage setings on a project (please give pointer to Photon Wiki without adding details of Photon). Hence pls., remove mention of user configuring datastores.
1. Photon resource tickets are deprecated in favor of quotas. Hence prefer not to mention resource tickets and instead point to Photon wiki (https://github.com/vmware/photon-controller/wiki).
1.